src/workloads/query/ProjectParse.yml

```diff
@@ -0,0 +1,91 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  Test $project parsing performance for small and large number of fields.
```

Comment:
Could you add keywords in addition to the description? For example: https://github.com/mongodb/genny/blob/1dbcd0ba8b678721774787c31310fa4ec0a522fd/src/workloads/sharding/DistinctCommands.yml#L14

We will use those for general workload search.

Reply:
Also, could you include a little more context. For example this text from your workload form is wonderful context for this workload 

> For large $projects it could take hours to parse previously. A good expected value would be milliseconds for each run.

Also, what was it about the $project that previously made it so slow? Feel free to link into relevant server tickets as well.

Reply:
Sounds good! I can add keywords and more details in the description.

Reply:
Thank you